src/components/Card/Card.tsx

```
@@ -0,0 +1,32 @@
+import { Label } from '../../Label/Label';
+import Badge from '../Badge/Badge';
+import type { CardProps } from './Card.types';
+
+const Card = ({ badges, view, date, writer, header, children }: CardProps) => {
+  return (
+    <article className="bg-white rounded-xl border border-gray-200 p-6 hover:border-purple-300 hover:shadow-lg transition-all text-left group">
+      <header className="flex items-center gap-2 mb-3 flex-wrap">
+        {badges.map((badge) => (
+          <Badge size="sm" variant={badge.variant}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)